CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # mdn-bcd-collector Changelog
 
+## v6.1.6
+
+### Test Changes
+
+#### Added
+
+- api.FileSystemDirectoryHandle.@@asyncIterator
+- api.ReadableStream.@@asyncIterator
+
+#### Removed
+
+- api.FileSystemDirectoryHandle.@@iterator
```

Comment:
These removals all make sense, these tests fail in Chrome, and presumably all browsers.